src/Compilers/Core/Portable/PEWriter/CustomDebugInfoWriter.cs

```diff
@@ -83,14 +83,6 @@ public byte[] SerializeMethodDebugInfo(EmitContext context, IMethodBody methodBo
             var pooledBuilder = PooledBlobBuilder.GetInstance();
             var encoder = new CustomDebugInfoEncoder(pooledBuilder);
 
-            // NOTE: This is an attempt to match Dev10's apparent behavior.  For iterator methods (i.e. the method
```

Comment:
The "unusual" behavior in ISymUnmanagedWriter has been fixed a while ago.  I don't think we need to document what Dev10 does anymore either.